debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlChangeRecordEmitter.java

```diff
@@ -27,16 +28,14 @@ public class MySqlChangeRecordEmitter extends RelationalChangeRecordEmitter<MySq
     private final OffsetContext offset;
     private final Object[] before;
     private final Object[] after;
-    private final boolean skipMessagesWithoutChange;
 
     public MySqlChangeRecordEmitter(MySqlPartition partition, OffsetContext offset, Clock clock, Operation operation, Serializable[] before,
-                                    Serializable[] after, boolean skipMessagesWithoutChange) {
-        super(partition, offset, clock);
+                                    Serializable[] after, CommonConnectorConfig connectorConfig) {
```

Comment:
Is it possible to use `MySqlConnectorConfig` here and the appropriate config types at other places too use the most specific type?